src/ui/components/ObjectFrame.vue

```diff
@@ -144,6 +149,11 @@ export default {
   computed: {
     statusClass() {
       return this.status ? `is-status--${this.status}` : '';
+    },
+    supportsIndependentTime() {
+      // const viewKey = this.getViewKey();
+
+      return true; //this.domainObject && SupportedViewTypes.includes(viewKey);
```

Comment:
Probably switch back to checking the viewKey? :)

Reply:
gotcha!